assets/js/script.js

```diff
@@ -0,0 +1,91 @@
+var grid = Array(100).fill(0)
+
+//store snakes and ladders keys. jump refers to change of position
+var jumps = {
+  1: 38,
+  4: 14,
+  9: 31,
+  17: 7,
+  21: 42,
+  28: 84,
+  51: 67,
+  54: 34,
+  62: 19,
+  64: 60,
+  72: 91,
+  80: 99,
+  87: 36,
+  93: 73,
+  95: 75,
+  98: 79,
+}
+
+function getJump(rollValue) {
+  if (jumps[rollValue]) {
```

Comment:
maybe update to
if key is valid, return value.
instead of if value is valid, return value. Both works the same though

Reply:
I don't think so. This function is for checking jump information in jump object.